packages/design-system/src/icons/icon-download.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
+import { IconBase, IconElement, IconProps } from './icon-base';
+
+export const IconDownload = React.forwardRef<IconElement, Readonly<IconProps>>(
+  ({ ...props }, forwardedRef) => (
+    <IconBase ref={forwardedRef} {...props} width={18} height={18}>
```

Comment:
Any reason the default width and height is 18?

Reply:
nah, it was just some value tweaking, now we have the correct svg we don't need these props anymore